### ISSUE TITLE
Cosmetic use history, facial

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9873]
**Preferred term label:**
Cosmetic use history, facial

**Synonyms**
Use of makeup on face

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Canaliculitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000297
- label: History of facia cosmetic use
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:28722951
- def: A medical history of use of facial makeup.
- comment: Facial makeup use may be associated with canaliculitis, which refers to inflammation of the lacrimal canaliculus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9873
- synonyms: Use of makeup on face(NONE)

**Comment by MickeySegal** [comment]
Typo: **facia** cosmetic
